docs/developer-center/new-guides/quickstart.md

```diff
@@ -1,185 +1,966 @@
+      <td>1185702.676</td>
+      <td>0.95</td>
+      <td>2f21749c02f73116892eb3b6fd5d5738</td>
+      <td>POINT (-74.02744 40.63152)</td>
+    </tr>
+    <tr>
+      <th>5</th>
+      <td>Caesar's Bay Shopping Center</td>
+      <td>8973 Bay Parkway,Brooklyn, NY 11214</td>
+      <td>1148427.411</td>
+      <td>0.95</td>
+      <td>134c23973313802448365db6235783f9</td>
+      <td>POINT (-74.00098 40.59321)</td>
+    </tr>
+  </tbody>
+</table>
+</div>
+
+
+
+Done! Now that the stores are geocoded, you will notice a new column called `geometry` has been added. This column stores the geographic location of each store and it's used to plot each location on the map.
+
+You can quickly visualize your geocoded dataframes using the Map and Layer classes. Check the [visualization guide]() to know all the visualization possibilities.
+
+
+```python
 from cartoframes.viz import Map, Layer
 
-Map(Layer(bikeshare_df))
+Map(Layer(starbucks_df))
 ```
 
-![Bikeshare data](../img/new-guides/quickstart/explore_layer.png)
+<div class="example-map">
+    <iframe
+        id="quickstart_guide_starbucks_brooklyn"
+        src="https://cartoframes.carto.com/kuviz/f442503a-f612-403d-b171-95e9138d0188"
+        width="100%"
+        height="500"
+        style="margin: 20px auto !important"
+        frameBorder="0">
+    </iframe>
+</div>
+
 
 Great! We have a map!
 
-Now, that you have a better sense of about where the stations are located, it's time to continue with the exploration. The next question to answer is which stations around the city are most active. To visualize this on the map, you can use a CARTOframes layer helper. Using the column `total_events`, which is the number of dropoffs and pickups at each station for the past month, let’s try a visualization helper called `size_continuous_layer` to size each station by that value:
+Now, you have a better sense about where the stores are. To continue with your exploration, you want to know where are the stores with more revenue. To do so, you can use the `size_continuous_layer` visualization layer:
+
 
-```py
+```python
 from cartoframes.viz.helpers import size_continuous_layer
 
-Map(size_continuous_layer(bikeshare_df, 'total_events'))
+Map(size_continuous_layer(starbucks_df, 'revenue', 'Revenue in $'))
 ```
-![Bikeshare data](../img/new-guides/quickstart/explore_helper.png)
 
-Good job! Now, just taking a look, you can see where are the stations with more activity. Also, thanks to be using a helper, we get a legend out of it.
+<div class="example-map">
+    <iframe
+        id="quickstart_guide_starbucks_brooklyn_by_revenue"
+        src="https://cartoframes.carto.com/kuviz/2e27384e-0cb4-4c02-809f-efa618ee5a6c"
+        width="100%"
+        height="500"
+        style="margin: 20px auto !important"
+        frameBorder="0">
+    </iframe>
+</div>
+
+Good job! Now, just taking a look, you can see where are the stores with more revenue. Also, thanks to be using a helper, we get a legend out of it.
```

Comment:
replace: "Now, just taking a look, you can see where are the stores with more revenue. Also, thanks to be using a helper, we get a legend out of it."

with 

"By using the [size continuous visualization layer](link to example) you can see right away where the stores with higher revenue are. By default, visualization layers also provide a popup with the mapped value and an appropriate legend."